doc/edgeconnect-api-ref.md

```diff
@@ -0,0 +1,35 @@
+EdgeConnect is the Schema for the EdgeConnect API
+
+## EdgeConnect is the Schema for the EdgeConnect API
```

Comment:
duplicate + does not say much

`EdgeConnect Schema` would be enough

Reply:
agree on it.

Reply:
fixed in 8eca23cd